numpy/_core/numerictypes.py

```diff
@@ -113,12 +113,10 @@
 generic = allTypes['generic']
 
 genericTypeRank = ['bool', 'int8', 'uint8', 'int16', 'uint16',
-                   'int32', 'uint32', 'int64', 'uint64', 'int128',
-                   'uint128', 'float16',
-                   'float32', 'float64', 'float80', 'float96', 'float128',
-                   'float256',
-                   'complex32', 'complex64', 'complex128', 'complex160',
-                   'complex192', 'complex256', 'complex512', 'object']
+                   'int32', 'uint32', 'int64', 'uint64', 'int128', 'uint128',
+                   'float16', 'float32', 'float64', 'float96', 'float128',
```

Comment:
Does these need float80/complex160 if you add it above?

Reply:
I actually wanted to remove those too, and ws in the middle of doing so. But after seeing it pop up all over the place I thought I'd "restore" them here. So yea this wasn't intentional